ehrapy/plot/supervised/_feature_importances.py

```diff
@@ -0,0 +1,30 @@
+import matplotlib.pyplot as plt
+import pandas as pd
+import seaborn as sns
+from anndata import AnnData
+
+
+def feature_importances(adata: AnnData, key: str = "feature_importances", n_features: int = 10):
+    """
+    Plot features with greates absolute importances as a barplot.
```

Comment:
This should be moved up into the first line of the docstring -> `"""Plot features bla.`